cart/datastores.bal

```diff
@@ -75,19 +77,43 @@ public isolated class InMemoryStore {
 }
 
 public isolated class RedisStore {
-    //TODO impl redis store
     *DataStore;
-    private map<Cart> store = {};
+    private final redis:Client redisClient;
 
-    isolated function add(string userId, string productId, int quantity) {
+    function init() returns error? {
+        self.redisClient = check new ({
+            host: redisHost,
```

Comment:
how do we take this value?

Reply:
This value is defined as configurable value https://github.com/ballerina-guides/gcp-microservices-demo/blob/9ba420ee9fdab329360b73acf2813c2b632abfba/cart/cart_service.bal#L21

Reply:
ah yes.